lib/gemaker/templates/engine/Gemfile

```
@@ -4,3 +4,8 @@ source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 gemspec
+
+papertrail_config_engine_path = Dir[
+  File.join(File.dirname(__FILE__)[/^.*(engines)/], "**", "papertrail_config")
+].first
```

Joven, revisé esto y creo que se puede acortar un poquito para dejarlo más simple.
Aparte, me parece que `Dir.glob` me dice más que `Dir[]`, por lo que creo que queda más claro.

```suggestion
papertrail_config_path = Dir.glob(File.join(__FILE__[/^.*engines/], "**", "papertrail_config"))[0]
```

Qué opinas?

---

Cambiado! La verdad es que no soy fan del `[0]` vs hacer `first`, pero me gusta que quede en 1 línea (y con el slice queda en 99 chars jaja)

---

Jajjaa, sí. Fue solo para que quedara en una línea y fuera más fácil de leer 😬